src/chttpd/src/chttpd_exclude_headers.erl

```diff
@@ -0,0 +1,45 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(chttpd_exclude_headers).
+-include_lib("couch/include/couch_db.hrl").
+
+-export([maybe_exclude_headers/2]).
+
+
+
+% List of essential headers for an http request. If we leave out server then Mochiweb adds in its default one
+-define(ESSENTIAL_HEADERS, ["Cache-Control", "ETag", "Content-Type", "Content-Length", "Vary", "Server"]).
```

Comment:
and probably best to lay them out one line per header, so we can modify it over time.